src/transformers/models/grounding_dino/processing_grounding_dino.py

```diff
@@ -56,6 +64,26 @@ def get_phrases_from_posmap(posmaps, input_ids):
     return token_ids
 
 
+class GroundingDinoProcessorKwargs(ProcessingKwargs, total=False):
```

Comment:
Should extend the default `ImagesKwargs` in the `image_kwargs` attribute in `ProcessingKwargs` as the default kwargs don't include `annotations`, `do_convert_annotations`, etc. In `GroundingDinoImageProcessor.preprocess`

Reply:
@EduardoPach I added additional kwargs that used in `GroundingDinoImageProcessor.preprocess` into ImagesKwargs. Does it seems okay?

Reply:
https://github.com/huggingface/transformers/pull/31368#issuecomment-2246659522

I think we can just put the additional value such as `annotations` in the calling processor.